en/telemetry/crsf_telemetry.md

```diff
@@ -0,0 +1,211 @@
+# CRSF Telemetry (TBS Crossfire Telemetry)
+
+CRSF is a telemetry protocol that can be used for both RC control and to get telemetry information from the vehicle/flight controller on a compatible RC transmitter.
+
+The protocol was developed by [Team BlackSheep](https://www.team-blacksheep.com/) for their Crossfire RC system, but is also used by [ExpressLRS (ELRS)](https://www.expresslrs.org/) RC systems.
+It is a bidirectional protocol that only needs a single UART for communicating both RC and telemetry.
+
+The [supported telemetry messages are listed here](#telemetry-messages), and include: flight mode, battery level, GPS data RC signal strength, speed, altitude, and so on.
+
+:::note
+If you don't need telemetry you can connect a TBS Crossfire to the `RCIN` port and configure the receiver to use S.BUS.
+Crossfire radio systems can also be used as [Telemetry Radios](../telemetry/README.md).
+:::
+
+:::warning
+PX4 does not include the CRSF protocol support by default.
+The [instructions below](#px4-configuration) explain how to build and upload custom PX4 firmware that includes the required modules.
+:::
+
+## Radio System Setup
+
+To use CRSF telemetry you will need a [TBS Crossfire radio system](#tbs-radio-systems) or [ExpressLRS radio system](#expresslrs-radio-systems) that includes an [RC controller](#rc-controllers) with a transmitter, and a receiver (from the same vendor).
+
+:::note
+An RC radio system historically consisted of a ground-based controller that transmitted to an on-vehicle receiver.
+Even though many radio systems are now bidirectional, the ground module may still be referred to as the transmitter, and the air unit may be called a receiver.
+:::
+
+Generally you will need to separately setup and configure the transmitter and receiver, and then _bind_ them together.
+
+A transmitter might come as an integral part of an [RC controller](#rc-controllers), or it might be a separate module that you plug into a controller.
+If it is a separate module then you may also need to update the module software on the transmitter to firmware that supports CRSF, such as OpenTX or EdgeTx.
+In both cases you will need to configure the transmitter to enable CRSF.
+
+The receiver must be [wired](#wiring) to a spare port (UART) on the Flight Controller.
+Then you can _bind_ the transmitter and receiver together.
+
+Instructions for the steps above are covered in
+
+- [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf)
+- [Express LRS: QuickStart](https://www.expresslrs.org/quick-start/getting-started/)
+
+### Wiring
+
+The TX and RX on your selected Flight Controller UART should be connected to separate channels on the receiver.
+The signal is usually uninverted, and can be directly connected (no additional inverter logic is required in the cable).
```

Comment:
IS this correct? And is it correct for both ELRS and TBS? I have covered my bases using the next sentence, which says "you should check"